### ISSUE TITLE
Sound is broken after installing Logitech G Hub software for headset

### ISSUE THREAD

**Issue by sines99** [BelledonneCommunications/linphone-desktop#317]
I am using a new headset. The Logitech PRO X gaming headset. After installing the Logitech G Hub software, the sound in Linphone is broken. If I don't use the headset's sound card, but the computer's, the sound is ok, but the microphone is not recognized. When I uninstall the Logitech software, the sound works fine again and the microphone is recognized. But then I hear myself twice. But that is not due to the Windows setting.

Why could the sound sound so broken when I set up the Logitech software? How can I fix it? Any ideas?

**Comment by ZaquMan** [comment]
Yes, this is an old issue, but I've been having the same issue, and the only resolution I found was to just uninstall Logitech G Hub.  It's not my desired solution, but seeing as it's been over two years, it might just be what it is for now.

**Comment by ZaquMan** [comment]
Update:
We've found that this happens with Corsair headsets when Corsair's iCue is installed.  Like with Logitech's G Hub, the solution I've found is to just uninstall iCue.  I think the issue comes from the management software's abilities to change the audio stream, such as changing EQ levels, but this isn't something I know how to dig into more.
